### ISSUE TITLE
libssl.so.1.1: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by mraymond** [Unleash/unleash-edge#509]
### Describe the bug

I'm attempting to set up a unleash-edge within a docker image built on a newer OS(Debian 12.6), and when starting the container, it returns the error
`/unleash-edge: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory`

Example Dockerfile, with latest binary(19.3.0) in same directory:
```
FROM debian:12.6

COPY ./ /

ENTRYPOINT ["/unleash-edge"]
```

### Steps to reproduce the bug

1. Setup Dockerfile using later versions of OSes(Debian 12) and unleash-edge binary in same folder
2. Run Dockerfile

### Expected behavior

Should start up unleash-edge without errors

### Logs, error output, etc.

```bash
/unleash-edge: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

### Unleash version

19.3.0

### Subscription type

Open source

### Hosting type

Self-hosted

**Comment by chriswk** [comment]
Hi @mraymond - With newer debians, debian uses openssl 3.0, I think we're still built against libssl (openssl 1.1). I'll see if I can't get a build out with openssl 3.0. Thanks for reporting this.

**Comment by chriswk** [comment]
I'm assuming you downloaded the binary from the repository here, that's still built with openssl 1.1.

I'd probably make a builder using debian 12 as well. I'll make a PR with a new Dockerfile with Openssl 3.0

**Comment by mraymond** [comment]
Yep, confirming the binaries I was trying were from the releases on Github. Thanks for the future update.
